pkg/kubelet/dockertools/manager.go

```diff
@@ -1181,7 +1185,7 @@ func (dm *DockerManager) killContainer(containerID types.UID) error {
 		}
 	}
 	dm.readinessManager.RemoveReadiness(ID)
-	err = dm.client.StopContainer(ID, 10)
+	err = dm.client.StopContainer(ID, gracePeriod)
```

Comment:
This is ok as an initial implementation, but what we really want is for the prestop hook execution time to be included in the grace period time budget.